inaugurator/main.py

```diff
@@ -44,6 +42,10 @@
 parser.add_argument("--inauguratorDontReadSmartData", action="store_true", default=False)
 parser.add_argument("--inauguratorDontFailOnFailedDisk", action="store_true", default=False)
 parser.add_argument("--inauguratorCleanupUpperPercentageThreshold", type=int, default=65)
+parser.add_argument("--inauguratorWipeOldInauguratorInstallations", action="store_true", default=False)
+parser.add_argument("--inauguratorTargetDeviceCandidate", nargs='+', help="This parameter is mutually exclusive with inauguratorTargetDeviceLabel and inauguratorTargetDeviceType")
+parser.add_argument("--inauguratorTargetDeviceLabel", default="", help="This parameter is mutually exclusive with inauguratorTargetDeviceCandidate and inauguratorTargetDeviceType")
```

Comment:
@eliran-stratoscale inauguratorExpectedLabel is different usage from  inauguratorTargetDeviceLabel